tests/acceptance/features/bootstrap/UserLdapGeneralContext.php

```diff
@@ -318,6 +320,118 @@ public function getLdapPort() {
 		return $this->ldapPort;
 	}
 
+	/**
+	 * @param $setDefaultAttributes
+	 * @param $table
+	 *
+	 * @return array
+	 */
+	public function setAttributesForUser($setDefaultAttributes, $table) {
+		$bodies = [];
+		foreach ($table as $row) {
+			$body['userid'] = $this->featureContext->getActualUsername($row['username']);
+
+			if (isset($row['displayname'])) {
+				$body['displayName'] = $row['displayname'];
+			} elseif ($setDefaultAttributes) {
+				$body['displayName'] = $this->featureContext->getDisplayNameForUser($body['userid']);
+				if ($body['displayName'] === null) {
+					$body['displayName'] = $this->featureContext->getDisplayNameForUser('regularuser');
+				}
+			} else {
+				$body['displayName'] = null;
+			}
+
+			if (isset($row['email'])) {
+				$body['email'] = $row['email'];
+			} elseif ($setDefaultAttributes) {
+				$body['email'] = $this->featureContext->getEmailAddressForUser($body['userid']);
+				if ($body['email'] === null) {
+					$body['email'] = $row['username'] . '@owncloud.org';
+				}
+			} else {
+				$body['email'] = null;
+			}
+
+			if (isset($row['password'])) {
+				$body['password'] = $this->featureContext->getActualPassword($row['password']);
+			} else {
+				$body['password'] = $this->featureContext->getPasswordForUser($row['username']);
+			}
+
+			// Add request body to the bodies array. we will use that later to loop through created users.
+			\array_push($bodies, $body);
+		}
+		return $bodies;
+	}
+
+	/**
+	 * @Given these users have been created using LDAP server:
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws LdapException
+	 * @throws Exception
+	 */
+	public function followingUsersHaveBeenCreatedWithoutSkeletonFilesUsingLDAPServer($table) {
+		$table = $table->getColumnsHash();
+		$setDefaultAttributes = '';
```

Comment:
```suggestion
		$setDefaultAttributes = false;
```
I think it would be better to use this variable as a boolean.